### PR TITLE
Make testharness.js output something in SVG documents

### PR DESCRIPTION
**Pull request by gsnedders**
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/8137)

### CODE REVIEW

File: resources/testharness.js [hunk 1]
```diff
@@ -2258,12 +2258,28 @@ policies and contribution forms [3].
...
-            node = output_document.createElement("div");
+            node = output_document.createElementNS("http://www.w3.org/1999/xhtml", "div");
             node.id = "log";
-            output_document.body.appendChild(node);
+            if (output_document.body) {
+                output_document.body.appendChild(node);
+            } else {
+                var is_svg = false;
+                try {
+                    is_svg = output_document.documentElement instanceof SVGSVGElement;
```
Comment (jgraham) [inline resources/testharness.js]: This doesn't work if `output_document != document`, right?

Comment (gsnedders) [reply resources/testharness.js]: @jgraham oh that's true

**Review by jgraham — APPROVED** [review]

File: resources/testharness.js (hunk 1 above)
Comment (gsnedders) [reply resources/testharness.js]: @jgraham fixed

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-11-09 22:35:13
Finished: 2017-11-09 22:39:44





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/20432)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/299890899)

**Review by jgraham — APPROVED** [review]

**Pull request merged**